src/transform/split-declaraion.js

```diff
@@ -0,0 +1,28 @@
+import traverser from '../traverser';
+import multiReplaceStatement from '../utils/multi-replace-statement';
+import VariableDeclaration from '../syntax/variable-declaration';
+
+export default function(ast) {
+  traverser.replace(ast, {
+    enter(node, parent) {
+      if (node.type === 'VariableDeclaration' && node.declarations.length > 1) {
+        this.skip();
+
+        splitDeclarations(parent, node);
+      }
+    }
+  });
+}
+
+function splitDeclarations(parent, node) {
```

Comment:
This function gets passed a single VariableDeclaration node that it splits. So it would be better named in singular form: `splitDeclaration()`.

Additionally it would be better to keep the arguments order consistent with the `enter()` above and in other functions in Lebab - so the actual node the function is acting on would be the first argument.